### PR TITLE
Ying wang

### PR DESCRIPTION
**Pull request by YingWANG14**

### CODE REVIEW

**Comment by YingWANG14** [comment]
Thanks for your advice! I’ll try them today!

On 02/16/2019 18:14，uhmwpe<notifications@github.com><mailto:notifications@github.com> wrote：

@uhmwpe commented on this pull request.

So overall, this was very well done. The only comments I have to add is that if you want to improve performance on your algorithm, you can utilize other methods such as nn.Dropout() and nn.BatchNorm(). Based on my personal experience, I also think nn.ReLU() is a preferable activation function to sigmoid. Finally, you can add more hidden layers simply by applying nn.Linear(hidden_dim, hidden_dim).
